README.md

```diff
@@ -81,7 +81,7 @@ module.exports = {
     },
     {
       files: ['**/*.gts'],
-      parser: 'eslint-plugin-ember/gjs-gts-parser',
+      parser: 'ember-eslint-parser',
```

Comment:
Can you make sure https://github.com/NullVoxPopuli/ember-eslint-parser has a helpful README?

Reply:
here ya go: https://github.com/NullVoxPopuli/ember-eslint-parser/pull/11